docs/widgets/calendar.rst

```diff
@@ -74,6 +74,12 @@ looks like ``const char * day_names[7] = {"Su", "Mo", ...};`` Only the
 pointer of the day names is saved so the elements should be static,
 global or constant variables.
 
+Custom year list
+----------------
+
+Sets a custom year list with :cpp:expr:`lv_calendar_set_year_list(calendar, years_list)`
+where ``years_list`` is a pointer to the custom years list ``static const char * years = "2023\n2022\n2021\n2020\n2019";``.
```

Comment:
```suggestion
where ``years_list`` is a pointer to the custom years list. It can be a constant string  like ``static const char * years = "2023\n2022\n2021\n2020\n2019";``, or can be generated dynamically into a buffer as well.
```

Reply:
Done in 0ff14d8